resources/views/layout/metadata.blade.php

```diff
@@ -16,7 +16,7 @@
 <link rel="manifest" href="{{ $appUrl }}/site.webmanifest">
 <link rel="mask-icon" href="{{ $appUrl }}/images/favicon/safari-pinned-tab.svg" color="#e2609a">
 <meta name="msapplication-TileColor" content="#603cba">
-<meta name="theme-color" content="hsl({{ $currentHue }}, 10%, 40%)"> {{-- @osu-colour-b1 --}}
+<meta name="theme-color" content="{{ hsl_to_hex($currentHue, 40, 50) }}"> {{-- @osu-colour-b1 --}}
```

Comment:
that's not `b1`?

Reply:
I'm not sure about that. It's a bit light and chrome actually set the text/icon colour to black for wiki section.

Reply:
hmm, i'll just revert to `b1` for now